DEPLOYMENT.md

```diff
@@ -0,0 +1,49 @@
+# [DRAFT] Deployment of workshoppers
+
+Suggesting steps to organize the process of publishing changes from workshoppers to npm.
+
+## Workshoppers versions
+
+Below, you can find a dashboard monitoring differences in versions of workshopper packages between github and npm.
+
+workshopper | NPM version   | Master of workshopper | Build | NodeJs LTS compatible |
+------------| ----------- | --------------------- | ----- | ---- |
+learnyounode | 3.5.10 | [3.5.10](https://github.com/workshopper/learnyounode) | [![Build Status](https://travis-ci.org/workshopper/learnyounode.svg?branch=master)](https://travis-ci.org/workshopper/learnyounode) | ✗
+javascripting | 2.6.1 | [2.6.1](https://github.com/workshopper/javascripting) | [![Build Status](https://travis-ci.org/workshopper/javascripting.svg?branch=master)](https://travis-ci.org/workshopper/javascripting) | ✗
+
+## Requirements
+
+Please be kind to prepare the following prerequisites before asking or publishing your package to npm:
+
+- New version of the package should follow semantic versioning
```

Comment:
I was imagining making a release in the github repository. ([example](https://gist.github.com/kalinchernev/1baab65db80984430893754af17def8e))
The release notes can be included manually while creating a draft or automatically if the commits included follow a convention.